webpack/webpack.config.babel.js

```diff
@@ -21,6 +66,26 @@ module.exports = {
 
   module: {
     preLoaders: [
+      // Replace HTML comments
+      {
+        test: /\.html$/,
+        exclude: /node_modules/,
+        loader: StringReplacePlugin.replace({
+          replacements: [
+            {
+              pattern: /<!--\sBOOTSTRAP-HTML\s-->/g,
+              replacement: function (match) {
+                return '<div id="canvas">' +
```

Comment:
#nit Could we make this like so to follow our style elsewhere and make it more readable?
```js
return (
  '<div id="canvas">' +
    '<div class="application-loading-indicator container ' +
      'container-pod vertical-center">' +
      bootstrap.HTML +
    '</div>' +
  '</div>'
);
```